templates/rally-beyond-the-paywall/measurements/advertisement.1.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$comment": "Please do not edit this schema by hand. Generate it instead by running npm run build:schema.",
+  "mozPipelineMetadata": {
+    "expiration_policy": {
+      "delete_after_days": 558
+    }
+  },
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "the type of event recorded. Either WebScience.advertisements, WebScience.articleContents, WebScience.pageNav, or WebScience.pageNavSensitive",
+      "enum": [
+        "WebScience.advertisements",
+        "WebScience.articleContents",
+        "WebScience.pageNav",
+        "WebScience.pageNavSensitive",
+        "WebScience.totalTiming"
+      ]
+    },
+    "userId": {
+      "type": "string",
+      "description": "a unique ID associated with a participant."
+    },
```

Comment:
Is this ping being sent via the glean.js or legacy pipeline. If the legacy pipeline, how does this differ from the rally/pioneer id?

Reply:
This should be the legacy pipeline.  This identifier is being used to join the Qualtrics survey data to study data and has been approved by data stewards.